src/btree/bt_import.c

```diff
@@ -23,10 +23,10 @@ __wt_import(WT_SESSION_IMPL *session, const char *uri)
 	WT_DECL_ITEM(checkpoint);
 	WT_DECL_RET;
 	WT_KEYED_ENCRYPTOR *kencryptor;
-	const char *filename;
+	char *checkpoint_list, *fileconf, *metadata, fileid[64];
 	const char *filecfg[] = {
 	   WT_CONFIG_BASE(session, file_meta), NULL, NULL, NULL, NULL, NULL };
-	char *checkpoint_list, *fileconf, *metadata, fileid[64];
+	const char *filename;
```

Comment:
This was unexpected, I would have expected `const type` to sort either after or before `type`, rather than inter-mixed, but if you think it's right, fbm.

Reply:
In `function.py`, `const`, `static` and `volatile` get stripped before applying the sort order.

Reply:
In retrospect, that seems wrong, but I'm probably the person that wrote it that way.

@agorrod, should we group `const` variables together, or inter-mix them with non-const variables?

Reply:
I feel like they should not be inter-mixed.

Reply:
@tetsuo-cpp, I know it's a pain, but maybe we could sort based on the type and then re-sort based on the leading static/const/volatile?

I haven't thought about this a lot, but that feels right? Types grouped together, and a secondary sort including the qualifiers?

Reply:
2e77263